FATEL_Backend/API/Controllers/ItemController.cs

```diff
@@ -20,7 +20,7 @@ public ItemController(IItemService itemService)
     }
     
     [HttpGet]
-    [Route("Read")]
+    [Route("Read/{id}")]
     public ActionResult<Item> Read(int id)
```

Comment:
```suggestion
    public ActionResult<Item> Read([FromRoute] int id)
```
To definitely bind the id to the route